lib/assets/assets.go

```diff
@@ -1,4 +1,3 @@
-// Package assets is generated by "lib/assets/generate"
```

Comment:
Can you remove this change? I have fixed the master, ypu don't need it anymore.

Reply:
strange because I have already removed it. See [here](https://github.com/alexferrari88/rod/blob/bug/631/lib/assets/assets.go).

Reply:
I rebased with master, solved the conflict, and now it should be as you said.